CHANGELOG.md

```diff
@@ -6,15 +6,18 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+* Add to `BakeNumberedNotes` directions for baking exercises in a note
 * Add `BakeEquations` direction (minor)
 * Remove `NoteElement` logging (minor)
-* Adds `BakeChapterReviewExercises` v2 and a u-physics `Strategy` to `BakeChapterAnswerKey` to support baking exercises in u-physics (minor)
+* Adds `BakeChapterReviewExercises` v2 and a u-physics `Strategy` to `BakeChapterAnswerKey` to support baking exercises in u-physics (minor) (with this comes removing bake checkpoints and theorems)
```

Comment:
tiny thing, but I believe this addition is actually to the `BakeNumberedNotes` change on line 9 above